tests/corsika/test_corsika7id.py

```diff
@@ -23,17 +23,20 @@ def test_from_pdgid():
     assert Corsika7ID.from_pdgid(PDGID(-13)) == 5
     assert Corsika7ID.from_pdgid(PDGID(13)) == Corsika7ID(6)
 
-
-def test_from_pdgid_non_matching():
```

Comment:
I think it is better to keep tests separate when they test different things. Also, a single test def should not do many tests as otherwise it would become difficult to know what is failing (I know, we are far from those crazy situations).

BTW, instead of changing to the assert as below you can simply assign to an unamed variable for the purpose of catching the exception, so `_ = Corsika7ID.from_pdgid(55)`.